test/BarcodeContainer.test.tsx

```diff
@@ -1,23 +1,23 @@
-/* eslint-env mocha */
-import React from 'react';
-import sinon from 'sinon';
-import { expect } from 'chai';
-import { mount } from 'enzyme';
+/** eslint-disable */
+import React from "react";
+import sinon from "sinon";
+import { mount } from "enzyme";
 // Import the related functions
-import CookieUtils from './../src/utils/CookieUtils.js';
+import CookieUtils from "./../src/utils/CookieUtils.js";
 // Import the component that is going to be tested
-import BarcodeContainer from './../src/components/BarcodeContainer/BarcodeContainer';
+import BarcodeContainer from "./../src/components/BarcodeContainer/BarcodeContainer";
 // Import mock data
-import mockBarcodeContainerTestData from './mockBarcodeContainerTestData.js';
+import mockBarcodeContainerTestData from "./mockBarcodeContainerTestData.js";
 
-describe('BarcodeContainer', () => {
-  describe('Before making API calls for barcodes, <BarcodeContainer>', () => {
+describe.skip("BarcodeContainer", () => {
```

Comment:
why is this skipped?

Reply:
The `BarcodeContainer` is a legacy component which didn't make it to production and I didn't want to just remove the component or tests in case we can reuse it soon (but still waiting to hear back on it).